### PR TITLE
SPEC-47 Integrate Eclipse Javamail - upgrade to 1.6.3

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins test please

File: appserver/core/api-exporter-fragment/pom.xml [hunk 1]
```diff
@@ -149,11 +149,11 @@
...
-                                    javax.mail; \
-                                    javax.mail.event; \
-                                    javax.mail.internet; \
-                                    javax.mail.search; \
-                                    javax.mail.util; \
+                                    jakarta.mail; \
+                                    jakarta.mail.event; \
+                                    jakarta.mail.internet; \
+                                    jakarta.mail.search; \
+                                    jakarta.mail.util; \
```
Comment (pdudits) [inline appserver/core/api-exporter-fragment/pom.xml]: We're not that far, are we? The packages are still the same.

Comment (Cousjava) [reply appserver/core/api-exporter-fragment/pom.xml]: No, this is the result of me doing a sed and not thinking

**Comment by Cousjava** [comment]
Jenkins test please

**Comment by jGauravGupta** [comment]
`javax.mail.jar` refrenced in the following files : 

- appserver\admin\production_domain_template_web\src\main\resources\config\default-web.xml
- appserver\admin\production_domain_template\src\main\resources\config\default-web.xml
- appserver\admin\gf_template_web\src\main\resources\config\default-web.xml
- appserver\admin\gf_template\src\main\resources\config\default-web.xml
- appserver\appclient\client\acc-standalone\fixup.xml
- appserver\webservices\webservices-scripts\src\main\resources\glassfish\bin\wsdeploy
- appserver\webservices\webservices-scripts\src\main\resources\glassfish\bin\wsdeploy.bat
- appserver\webservices\webservices-scripts\src\main\resources\glassfish\bin\wscompile
- appserver\webservices\webservices-scripts\src\main\resources\glassfish\bin\wscompile.bat

File: appserver/javaee-api/javax.javaee-api/pom.xml [hunk 2]
```diff
@@ -125,13 +125,13 @@
             <optional>true</optional>
         </dependency>
         <!--
-            using com.sun.mail.java.mail instead of javax.mail:javax.mail-api
-            because javax.mail has a dependency to com.sun.mail classes,
+            using com.sun.mail.java.mail instead of jakarta.mail:jakarta.mail-api
+            because jakarta.mail has a dependency to com.sun.mail classes,
```
Comment (pdudits) [inline appserver/javaee-api/javax.javaee-api/pom.xml]: Is this comment yet true (after the replace?)

**Comment by Cousjava** [comment]
Jenkins test please

File: appserver/javaee-api/javax.javaee-api/pom.xml (hunk 2 above)
Comment (Cousjava) [reply appserver/javaee-api/javax.javaee-api/pom.xml]: yes

**Pull request merged**